db/migrate/20190416093524_add_aasm_state_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAasmStateToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :user, :aasm_state, :string
```

Comment:
Should this be nullable?

Reply:
No. The initial state set in the model covers that.